contracts/l2/token/GraphTokenUpgradeable.sol

```diff
@@ -50,6 +50,8 @@ contract GraphTokenUpgradeable is GraphUpgradeable, Governed, ERC20BurnableUpgra
     bytes32 private DOMAIN_SEPARATOR;
     mapping(address => bool) private _minters;
     mapping(address => uint256) public nonces;
+    // Storage gap added in case we need to add state variables to this contract
+    uint256[47] private __gap;
```

Comment:
We could just keep the gap at 50 until an upgrade happens. But it is nothing critical

Reply:
yeah, I chose 47 so that for both contracts we use a standard number of total used slots, i.e. 50